### PR TITLE
dep(both): update gomod to remove some tidb-tools lib into tidb

### PR DESCRIPTION
**Pull request by hawkingrei**
Signed-off-by: Weizhen Wang <wangweizhen@pingcap.com>

### What problem does this PR solve?

Issue Number: ref #5035

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation
 - Need to update key monitor metrics in both TiCDC document and official document

### Release note 

```release-note
Please add a release note.
If you don't think this PR needs a release note then fill it with `None`.
```

### CODE REVIEW

**Comment by lance6716** [comment]
CI says a package should not be replaced to tidb's prefix. @hawkingrei

**Comment by lance6716** [comment]
waiting https://github.com/pingcap/tidb/pull/33562

**Review by lance6716 — APPROVED** [review]

**Comment by hawkingrei** [comment]
> CI says a package should not be replaced to tidb's prefix. @hawkingrei

I think it need to add some leak ignoration such as ```log```.

**Review by ben1009 — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -48,15 +48,15 @@ require (
 	github.com/pingcap/check v0.0.0-20211026125417-57bd13f7b5f0
 	github.com/pingcap/errors v0.11.5-0.20211224045212-9687c2b0f87c
 	github.com/pingcap/failpoint v0.0.0-20220303073211-00fea37feb66
-	github.com/pingcap/kvproto v0.0.0-20220314103629-10e688307221
+	github.com/pingcap/kvproto v0.0.0-20220328072018-6e75c12dbd73
 	github.com/pingcap/log v0.0.0-20211215031037-e024ba4eb0ee
-	github.com/pingcap/tidb v1.1.0-beta.0.20220318092833-a30449081049
+	github.com/pingcap/tidb v1.1.0-beta.0.20220329110428-fbaaa1117936
 	github.com/pingcap/tidb-tools v6.0.0-alpha.0.20220317013353-dfc5146f4746+incompatible
```
Comment (overvenus) [inline go.mod:54]: Why is it still here?

**Comment by lance6716** [comment]
/hold

need to check if all libraries of tidb-tools can be removed. @hawkingrei can you help us?

File: go.mod (hunk 1 above)
Comment (hawkingrei) [reply go.mod:54]: in the tidb-tools, some libraries import tiflow. so it needs to update the tiflow and then migrate this library into tidb. They are waiting for this PR.

**Comment by lichunzhu** [comment]
> worker

Yes it will

**Comment by lance6716** [comment]
in my memory kafka-integration-test always fails, request for help @overvenus

**Comment by lance6716** [comment]
> > I'll check CI faillure soon. Will stop-relay wait for DM-worker finish stoping? @lichunzhu
> 
> Yes it will

Found it's still TODO

https://github.com/pingcap/tiflow/blob/63e06c907e6b554eaeea7c3a998785adcb562826/dm/dm/master/server.go#L2543-L2560

/unhold

**Pull request merged**